plugins/torch/plugin_test/test_torch.py

```diff
@@ -97,9 +93,9 @@ def test_fit(db, model):
     )
 
     model.trainer = trainer
-    model.validation = Validation(
-        'my_valid',
-        metrics=[Metric(identifier='acc', object=acc)],
-        datasets=[valid_dataset],
-    )
     db.apply(model)
+
+    captured = capfd.readouterr()
+
+    # check that the training actually happened
+    assert 'TRAIN; iteration:' in captured.out
```

Comment:
Can we overfit on data and do predict after db.load to assure the training has actually happend and the model is updated with trained model